pkg/logs/auditor/auditor.go

```diff
@@ -34,35 +35,35 @@ type Auditor struct {
 	inputChan    chan message.Message
 	registry     map[string]*RegistryEntry
 	registryPath string
-
-	entryTTL time.Duration
-
-	done chan struct{}
+	mu           sync.Mutex
+	entryTTL     time.Duration
+	done         chan struct{}
 }
 
 // New returns an initialized Auditor
 func New(inputChan chan message.Message, runPath string) *Auditor {
 	return &Auditor{
 		inputChan:    inputChan,
 		registryPath: filepath.Join(runPath, "registry.json"),
+		mu:           sync.Mutex{},
```

Comment:
The zero value for a Mutex is an unlocked Mutex, so you don't need to initialize it.